docs/development/pigs/pig-013.rst

```diff
@@ -15,19 +15,25 @@ PIG 13 - Gammapy dependencies and distribution
 Abstract
 ========
 
-As preparation for the planned Gammapy v1.0 release fall 2019, we propose to
-clarify which Gammapy dependencies are required and which ones are optional. We
-also propose to increase the minimum required version for some dependencies.
-Finally, we propose some updates to the Gammapy supported distribution channels
-and installation instructions.
-
-This represents that plan for 2019 to be implemented before v1.0. Things are
-likely to evolve in the coming years, as Gammapy supports different projects,
-especially CTA.
+This is a proposal to make some updates concerning Gammapy dependencies and
+supported distribution channels, to be introduced in Gammapy v0.14, and then to
+be used in Gammapy v1.0 and likely without much changes moving forward.
+
+Given that we have a good way to ship recent versions of Gammapy and all
+dependencies to users on all platforms, we suggest to drop support for Python
+3.5 or later, and to increase the minimum supported version of our dependencies.
```

Comment:
> we suggest to drop support for Python
3.5 or later, 

This reads a little bit ambiguous, do you mean to say to go for 3.5+ or 3.6+ or 3.7+ here?